features/FEATURE_LWIP/lwip-interface/lwip_stack.c

```diff
@@ -438,6 +453,7 @@ int lwip_bringup(bool dhcp, const char *ip, const char *netmask, const char *gw)
 
     netif_set_up(&lwip_netif);
 
+#if !DEVICE_EMAC
```

Comment:
Shouldn't the enclosed be a subroutine call to lwip_start_dhcp? Likewise the static address bit?